### PR TITLE
Es6

### PR DESCRIPTION
**Pull request by andrewdhazlett**
depends on https://github.com/jfols/liftoff/pull/5 and https://github.com/jfols/liftoff/pull/6

### CODE REVIEW

**Comment by jfols** [comment]
this looks beautiful, I'll test and fix conflicts tomorrow

File: server/fixtures.js [hunk 1]
```diff
@@ -1,24 +1,19 @@
...
+  let userCount = 50;
 
   console.log('FIXTURES: creating test users');
 
-  users = [];
-  _(userCount).times(function (n) {
-    users.push({
-      username: `${usernamePrefix}${n}`,
-      email: `${usernamePrefix}${n}${emailPostfix}`,
+  _.chain(userCount)
```
Comment (jfols) [inline server/fixtures.js]: I don't think you want to use `chain` here, this doesn't work

Comment (jfols) [inline server/fixtures.js]: How about this instead:

```
_.times(userCount, (index) => {
    Accounts.createUser({
      username: `${ usernamePrefix }${ index }`,
      email: `${ usernamePrefix }${ index }${ emailPostfix }`,
      password: 'password'
    });
  });
```

Comment (andrewdhazlett) [inline server/fixtures.js]: i think this is better anyway

**Pull request merged**